sklearn/utils/fixes.py

```diff
@@ -314,3 +316,22 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
     from ._scipy_sparse_lsqr_backport import lsqr as sparse_lsqr
 else:
     from scipy.sparse.linalg import lsqr as sparse_lsqr
+
+
+if sys.version < '2.7':
```

Comment:
Could you try `sys.version_info < (2, 7, 0)`, that seems to be more consistent across the three python versions :)